docs/source/cn/constant.py

```diff
@@ -630,7 +630,97 @@
 )
 
 reset_docstr(
-    oneflow.new_ones,
+    oneflow.utils.from_torch,
+    r"""
+    from_torch(input) -> Tensor
+
+    从 torch 张量新建一个 oneflow 张量。
+    返回的张量和 torch 张量共享内存。
+    
+    .. note::
+        目前仅支持 cpu 张量和本地张量。
+
+        此函数可被用于空间数据处理阶段和 torch 的一些 cpu 操作。
+
+    参数：
+        - **input** (torch.Tensor): 输入张量
+
+    返回值：
+        oneflow.Tensor
+
+    示例：
+
+    .. code-block:: python
+
+        import oneflow as flow
+        import torch
+
+        torch_t = torch.tensor([[1, 2, 3], [4, 5, 6]])
+        flow_t = flow.utils.from_torch(torch_t)
+    
+    此 ``from_torch`` 功能尚在测试阶段。
+    """
+
+
+)
+
+reset_docstr(
+    oneflow.utils.to_torch,
+    r"""
+    to_torch(input) -> Tensor
+
+    从 oneflow 张量新建一个 torch 张量。
+
+    返回的函数和 oneflow 张量共享内存。
+    
+    .. note::
+        目前仅支持 cpu 张量和本地张量。
+
+    参数：
+        - **input** (oneflow.Tensor): 输入张量
+
+    返回值：
+        torch.Tensor
+
+
+    示例：
+
+    .. code-block:: python
+
+        import oneflow as flow
+        import torch
+
+        flow_t = flow.tensor([[1, 2, 3], [4, 5, 6]])
+        torch_t = flow.utils.to_torch(flow_t)
+
+    此 ``to_torch`` 功能尚在测试阶段。
+    """
+)
+
+reset_docstr(
+    oneflow.nn.init.orthogonal_,
+    r"""
+    orthogonal_(tensor,gain)
```

Comment:
```suggestion
    orthogonal_(tensor, gain)
```

参数列表逗号之后要有空格，好看点，也是编码规范。